sdk/version.go

```diff
@@ -0,0 +1,18 @@
+package sdk
+
+import (
+	_ "embed"
+	"strings"
+
+	"github.com/blang/semver"
+)
+
+//go:embed .version
+var version string
+
+// Version is the release version of the SDK.
+var Version semver.Version
+
+func init() {
+	Version = semver.MustParse(strings.TrimSpace(version))
```

Comment:
@Frassle I like your idea of making this a blang/semver type. Are we ok with panicking if there's no .version file or do we want to fall back to an empty version?